libs/vkd3d/device.c

```diff
@@ -5508,6 +5508,24 @@ static HRESULT STDMETHODCALLTYPE d3d12_device_CheckFeatureSupport(d3d12_device_i
             return S_OK;
         }
 
+        case D3D12_FEATURE_D3D12_TIGHT_ALIGNMENT:
+        {
+            D3D12_FEATURE_DATA_TIGHT_ALIGNMENT *data = feature_data;
+
+            if (feature_data_size != sizeof(*data))
+            {
+                WARN("Invalid size %u.\n", feature_data_size);
+                return E_INVALIDARG;
+            }
+
+            /* Trivial vulkan catch-up, always supported */
+            data->SupportTier = D3D12_TIGHT_ALIGNMENT_TIER_1;
```

Comment:
I don't think we can expose this on devices with bufferImageGranularity that is larger than 4 KiB (i.e. older NV).

Reply:
Think we can't even rely on tier 1 since we might still create linear images for weird formats.